pkg/apis/app/v1beta1/application.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/kubesdk/pkg/component"
+	cr "sigs.k8s.io/kubesdk/pkg/customresource"
+	"sigs.k8s.io/kubesdk/pkg/finalizer"
+	"sigs.k8s.io/kubesdk/pkg/resource"
+	"sigs.k8s.io/kubesdk/pkg/status"
+	"strings"
+)
+
+// Mutate - mutate expected
+func (a *Application) Mutate(rsrc interface{}, labels map[string]string, status interface{}, expected, dependent, observed *resource.ObjectBag) (*resource.ObjectBag, error) {
+	return observed, nil
+}
+
+// Finalize - execute finalizers
+func (a *Application) Finalize(rsrc, sts interface{}, observed *resource.ObjectBag) error {
+	finalizer.Remove(a, finalizer.Cleanup)
+	return nil
+}
+
+//DependantResources - returns dependent rsrc
```

Comment:
Do you mean 'dependent', 'dependant', or both?  The latter is a legit British word.

Reply:
will change to "dependent". Had similar comment earlier as well. Guess i am still used to the British spelling.

Reply:
done